database/index.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-console */
```

Comment:
interesting, I'm not aware of this eslint config, I'm curious since mine isn't working

Reply:
Yeah, eslint tells me that I have unnecessary console.logs. It doesn't like them. I needed to keep some of the console.logs so I had to disable for those specific files so I don't see the annoying squiggly line from eslint lol